app/gui/src/dashboard/layouts/AssetContextMenu.tsx

```diff
@@ -89,8 +89,10 @@ export default function AssetContextMenu(props: AssetContextMenuProps) {
   const self = permissions.tryFindSelfPermission(user, asset.permissions)
   const path = asset.ensoPathValue
   const copyMutation = copyHooks.useCopy({ copyText: path ?? '' })
+  const copyIdMutation = copyHooks.useCopy({ copyText: asset.id })
```

Comment:
It looks like it's better instead of passing `copyText` as a prop, pass as a variable in mutate() instead